appveyor.yml

```diff
@@ -0,0 +1,16 @@
+clone_folder: C:\projects\neon-lang\
```

Comment:
There is a VERY good reason to have it there, as that is where our NEON-LANG project on AppVeyor is setup to drop our files into...  So, it's either we do a CD, or I pass --directory, which would you prefer?  I'd prefer the --directory, since it automagically tells SCons which folder to go to...